README.md

```diff
@@ -42,6 +42,12 @@ Drag and drop an element in another one.
 
 Example: `cy.getByTestAttr('draggable').dragAndDrop(cy.getByTestAttr('droppable'))`
 
+### state/dispatchActions
+
+Dispatch ngxs actions.
```

Comment:
FAR: explain that the store must be exposed in the global window object by the app, with an example:

```typescript
// app.component.ts
constructor(private readonly store: Store) {
  if ('Cypress' in window) {
    window.store = this.store;
  }
}
```

And explain this is a standard practice in Cypress, what are the benefits: https://docs.cypress.io/guides/references/best-practices#Organizing-Tests-Logging-In-Controlling-State